pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -60,6 +65,9 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if kc.HealthzPort != 0 && utilvalidation.IsValidPortNum(int(kc.HealthzPort)) != nil {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: HealthzPort (--healthz-port) %v must be between 1 and 65535, inclusive", kc.HealthzPort))
 	}
+	if !localFeatureGate.Enabled(features.CPUCFSQuotaPeriod) && (kc.CPUCFSQuotaPeriod != zeroDuration && kc.CPUCFSQuotaPeriod != metav1.Duration{Duration: 100 * time.Millisecond}) {
```

Comment:
nit: don't think we need the extra parens around `kc.CPUCFSQuotaPeriod != zeroDuration && kc.CPUCFSQuotaPeriod != metav1.Duration{Duration: 100 * time.Millisecond}`

Reply:
Yeah I put it there for my clarity. Removed it.